src/bot.js

```diff
@@ -15,14 +15,14 @@ class Bot {
 
 		// TODO: get list of rooms from |queryresponse|rooms to determine if
 		// they're public or not.
-		// this.roomsData = Set();
+		// this.roomsData = new Set();
```

Comment:
`Set` here is a set from immutable.js, not the sets from ES5, which doesn't use `new`. That's kind of confusing though. Could you replace cases where I use immutable `Set`s and `Map`s with `ISet` and `IMap`?